R/cluster.R

```diff
@@ -10,7 +10,12 @@
 #' @param BLUSPARAM A 
 #'   \code{\link[https://rdrr.io/github/LTLA/bluster/man/BlusterParam-class.html]{BlusterParam}}
 #'   object.
-#'
+#'   
+#' @param altExp A single character value indicating which alternate
```

Comment:
Is this "altExp" or "altexp" in the other functions?

We could use the @inheritParams to avoid duplication and divergence of documentation entries for common function arguments.

Reply:
It is altexp, my bad, I'll change that.